images/upgrade-mattermost-cloud-image.svg

```diff
@@ -0,0 +1,37 @@
+<svg width="234" height="143" viewBox="0 0 234 143" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Do we need to wrap these in components to make the svg theme compatible?

Reply:
Shouldn't be an issue as this is only on display in the System Console, which isn't themeable afaik.